panels/automated_emails.py

```diff
@@ -0,0 +1,6 @@
+from panels import *
+
+AutomatedEmail.extra_models[PanelApplication] = lambda session: session.query(PanelApplication).all()
+
+AutomatedEmail(PanelApplication, 'Your {EVENT_NAME} Panel Application Has Been Received', 'panel_app_confirmation.txt',
+               lambda app: True)
```

Comment:
This is the first time we've wanted to define an ``AutomatedEmail`` with literally no ``filter`` parameter.  Perhaps we should change ``filter`` to have a default value of ``lambda m: True``.